next-cloudinary/src/components/CldImage/CldImage.tsx

```diff
@@ -79,20 +85,36 @@ const CldImage = (props: CldImageProps) => {
   }
 
   async function handleOnError(options: HandleOnError) {
-    const result = await pollForProcessingImage({ src: options.target.src })
-    if ( result ) {
+    const result = await pollForProcessingImage({ src: options.target.src });
+    if (result) {
       setImgKey(`${defaultImgKey};${Date.now()}`);
     }
   }
 
+  function handleOnLoadingComplete(img: HTMLImageElement) {
+    if (props.onLoadingComplete) {
```

Comment:
i would recommend making these include in the if statement:

```
if ( typeof props.onLoad === 'function ) 
```

reason being, if for whatever reason someone passes in a value that is truthy but not a function, we'll get an error

Reply:
Will TypeScript not take care of this  since the type for that handler is already marked as a function?